docs/plugins/client-crud-plugin.md

```diff
@@ -1,7 +1,39 @@
 ---
-id: client-crud
+id: crud-client
 title: ClientCRUD Plugin
 sidebar_label: ClientCRUD
 ---
 
-## TODO
\ No newline at end of file
+The `ClientCRUDPlugin` plugin creates your GraphQL schema with all input types, Query, Mutation and Subscription fields following recommended patterns from [GraphQL CRUD](https://graphqlcrud.org/).
+The plugin also creates your CRUD resolvers to use with your GraphQL schema.
+
+## Installation
+
+```bash
+yarn add @graphback/codegen-schema
+```
+
+## Usage
+
+To use the plugin, add it to the `plugins` array in [`buildGraphbackAPI`](../api/build-graphback-api#plugins).
+
+```ts
+const { schema, resolvers } = buildGraphbackAPI(modelDefs, {
+  dataProviderCreator: createKnexDbProvider(db),
+  plugins: [
+    // highlight-start
+    new SchemaCRUDPlugin({
```

Comment:
```suggestion
    new ClientCRUDPlugin({
```

?

Reply:
Draft PR, ignore it.

Reply:
Generally speaking you can ignore draft PRs unless I ping you :smile:

Reply:
Lol I did not see that it is a draft.. realised it when I finished reviewing the first page ha ha Sorry for the noise ;-)